src/main/java/org/jabref/model/paging/Page.java

```diff
@@ -0,0 +1,41 @@
+package org.jabref.model.paging;
+
+import java.util.Collection;
+import java.util.Collections;
+
+public class Page<T> {
+
+    private int pageNumber;
+    private String query;
+    private Collection<T> content;
+
+    public Page(String query, int pageNumber, Collection<T> content) {
+        this.query = query;
+        this.pageNumber = pageNumber;
+        this.content = content;
+    }
+
+    public Page(String query, int pageNumber) {
+        this(query, pageNumber, Collections.emptyList());
+    }
+
+    public Collection<T> getContent() {
+        return content;
+    }
+
+    public void setContent(Collection<T> content) {
```

Comment:
I would not have a page modifiable.

See "Item 17: Minimize mutability" in "[Effective Java](https://www.bookzilla.de/shop/article/38214323/joshua_bloch_effective_java.html)"

Reply:
You're right, i forgot to remove the setter when i removed the posibility to search by a Page